frontend/src/components/GamePage/GameFooter.vue

```diff
@@ -22,9 +22,15 @@
 import Toolbar from 'primevue/toolbar'
 import Dialog from 'primevue/dialog'
 import Button from 'primevue/button'
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
+import { useArtStore } from '../../stores/art.ts'
 
-const combo = ref(0)
+// const combo = ref(0)
+const artStore = useArtStore()
+const combo = computed(() => {
```

Comment:
computed doesn't need to be used like this
https://pinia.vuejs.org/core-concepts/#Destructuring-from-a-Store